requirements.txt

```diff
@@ -7,3 +7,5 @@ pydantic==1.6.1
 SQLAlchemy==1.3.20
 starlette==0.13.6
 uvicorn==0.12.1
+hashids==1.3.1
+requests==2.25.1
```

Comment:
I needed to add requests for tests, is there a nicer way to handle those in python?